client/src/pages/NeedHelp.js

```diff
@@ -160,6 +161,13 @@ const NeedHelp = withRouter((props) => {
     setState({ ...updatedAnswers });
     if (key === "email") {
       localStorage.setItem("needHelpAnswers", JSON.stringify(updatedAnswers));
+      if (value) {
```

Comment:
L64 will check if the value is not falsy, if it is not then  we will make the API request, cuz Skip is being handled on submit and this will execute as well on skip.